tests/__mocks__/adapter.ts

```diff
@@ -4,47 +4,56 @@ import { Embedding } from "../../src/adapters/openai/helpers/embedding";
 import { STRINGS } from "./strings";
 
 export interface CommentMock {
-  id: number;
-  commentbody: string;
-  issuebody: string;
+  id: string;
+  plaintext: string | null;
+  author_id: number;
+  commentobject?: Record<string, unknown> | null;
```

Comment:
```suggestion
  commentObject?: Record<string, unknown> | null;
```

Our commit hook shouldn't have let this pass because we have strict pascal case and cspell

Reply:
I wanted to keep it consistent with the DB schema. This passed the cspell check because I had added it to the ignore list.

Reply:
> with the DB schema

If you defined it, then its wrong. If its from another database that we can't change, then we can add to the ignore list.